server/xpub-model/entities/user/helpers/elife-api.js

```diff
@@ -13,16 +13,20 @@ const request = (endpoint, query = {}) => {
   if (secret) {
     req.header.Authorization = secret
   }
-  return req
-    .query(query)
-    .catch(err => {
-      logger.error('Failed to fetch from eLife API:', err.message)
-      throw err
-    })
+  return req.query(query).catch(err => {
+    logger.error('Failed to fetch from eLife API:', err.message)
+    throw err
+  })
 }
 
 const convertPerson = apiPerson => {
-  const { id, name, research, emailAddresses, affiliations } = apiPerson
+  const {
+    id,
+    name = {},
```

Comment:
wasn't `name` considered mandatory?

Reply:
Can we have a unit test for `convertPerson`